octopusdeploy_framework/schemas/channel.go

```diff
@@ -72,14 +66,22 @@ func (c ChannelSchema) GetResourceSchema() resourceSchema.Schema {
 							Optional: true,
 						},
 					},
+					Blocks: map[string]resourceSchema.Block{
+						"action_package": resourceSchema.ListNestedBlock{
```

Comment:
Was there a reason for removing the required property here? I think action_package is required on the API